source.go

```diff
@@ -222,6 +253,24 @@ type SourceMandate struct {
 	URL                string                          `json:"url"`
 }
 
+// SourceSourceOrderItems describes the items on source orders for sources.
+type SourceSourceOrderItems struct {
+	Amount      int64                     `json:"amount"`
+	Currency    Currency                  `json:"currency"`
+	Description string                    `json:"description"`
+	Quantity    int64                     `json:"quantity"`
+	Type        SourceSourceOrderItemType `json:"type"`
+}
+
+// SourceSourceOrder describes a source order for a source.
+type SourceSourceOrder struct {
+	Amount   int64                  `json:"amount"`
+	Currency Currency               `json:"currency"`
+	Email    string                 `json:"email"`
+	Items    SourceSourceOrderItems `json:"items"`
```

Comment:
Mind making this one a pointer to `SourceSourceOrderItems`? It probably doesn't matter that much, but it's basically the overwhelming convention we have.

Reply:
Got it. Yeah I'd like to eventually get spun up on releasing. I have a few other projects I'm trying to wrap up before I'll be ready to learn that :)